docs/utils/immutable.md

```diff
@@ -30,9 +30,10 @@ class TodoStore {
   static displayName = 'TodoStore'
 
   constructor() {
-    this.state = {
-      todos: Immutable.Map({})
-    };
+    this.state = Immutable.Map({
+      todos: Immutable.List([]),
+      loading: false
```

Comment:
Yes `loading` was just meant to be an example of a random variable you might include in state.  Any suggestions to clarify this?